src/managers/ReactionManager.js

```diff
@@ -73,6 +73,7 @@ class ReactionManager extends BaseManager {
     const existing = this.cache.get(id);
     if (!this._partial(reactionEmoji)) return existing;
     const data = await this.client.api.channels(this.message.channel.id).messages(this.message.id).get();
+    if (this.message.partial) this.message.patch(data);
```

Comment:
```suggestion
    if (this.message.partial) this.message._patch(data);
```
`Message#patch` is strictly used for `MESSAGE_UPDATE`s (and doesn't achieve what is wanted here either), so `_patch` should be used here.

Reply:
ah, i couldnt find _patch before, maybe i was just blind :eyes: